src/modeles/listeAvis.js

```diff
@@ -0,0 +1,12 @@
+const Avis = require('./avis');
+const ElementsConstructibles = require('./elementsConstructibles');
+const Referentiel = require('../referentiel');
+
+class ListeAvis extends ElementsConstructibles {
+  constructor(donnees = {}, referentiel = Referentiel.creeReferentielVide()) {
+    const { avis = [] } = donnees;
```

Comment:
Ici nous utilisons avis et non listeAvis, je comprends pourquoi en lisant dossier.js mais cela peut entrainer une incompréhension. Je propose d'utiliser listeAvis et même si tristement cela entraine des modifications dans le toJSON de dossier (avis: this.listeAvis.toJSON())

Reply:
Pas forcément d'accord avec ça, on tombe vraiment mal en effet car `avis` ne change pas au pluriel, mais j'ai peur de compléxifier encore plus le modèle...